core/capabilities/registry.go

```diff
@@ -0,0 +1,63 @@
+package capabilities
+
+import (
+	"context"
+	"fmt"
+	"sync"
+)
+
+// Registry is a struct for the registry of capabilities.
+// Registry is safe for concurrent use.
+type Registry struct {
+	m  map[string]Capability
+	mu sync.RWMutex
+}
+
+// Get gets a capability from the registry.
+func (r *Registry) Get(_ context.Context, id string) (Capability, error) {
```

Comment:
Why include Context if it's unused? Is this trying to implement some interface..?

Reply:
Mainly because it's good practice and because I anticipate that we'll start adding more logic in here over time. At that point being able to hook into context.Context will be useful.